usaspending_api/api_contracts/contracts/award/Transactions.md

```diff
@@ -0,0 +1,68 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Transactions Page
+This is some description
```

Comment:
Suggested description: "This endpoint is used to power the Transaction History tables on USAspending.gov's award summary pages. This data can be used to better understand the details of specific transactions on a given award."